assets/templates/accounts/user_history_claimed.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'accounts/user_history_base.html' %}
+{% load staticfiles %}
+
+{% block title %}{{ request.user }} - Dining history {% endblock %}
+
+{% block tab_claimed %} active{% endblock %}
+
+{% block details %}
+    <div class="row">
+        <p class="col-12">
```

Comment:
`p` followed by `div` is fine, you'll just not find it in the doc because the doc focuses on single components and not on complete pages which are collections of different components.